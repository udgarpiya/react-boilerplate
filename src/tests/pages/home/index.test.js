import React from 'react'
import { shallow } from 'enzyme'
import Home from 'pages/home'

describe('Custom Head Component', () => {
  it('It should match snapshot', () => {
    const page = shallow(<Home />)
    expect(page).toMatchSnapshot()
  })
})
