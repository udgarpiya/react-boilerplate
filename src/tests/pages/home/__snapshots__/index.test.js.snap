// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Custom Head Component It should match snapshot 1`] = `
<Layout
  description=""
  title=""
>
  Home Page
</Layout>
`;
